src/vnsw/opencontrail-vrouter-netns/opencontrail_vrouter_netns/vrouter_hyperv.py

```diff
@@ -25,15 +25,98 @@ def powershell(cmds):
     return output
 
 
-class HyperVManager(object):
+LEFT_DEV_PREFIX = 'int'
```

Comment:
I'm not sure about this, since this is not configurable in vrouter_netns.py either